packages/core/ai_content_platform/vimax/agents/storyboard_artist.py

```
@@ -0,0 +1,194 @@
+            self.logger.info(
+                f"Generated storyboard: {len(images)} images, "
+                f"${total_cost:.3f} total cost"
+            )
+
+            return AgentResult.ok(
+                storyboard,
+                image_count=len(images),
+                cost=total_cost,
+            )
+
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)